pkg/controller/config/config_controller.go

```diff
@@ -93,13 +94,19 @@ func newReconciler(mgr manager.Manager, cm *cm.CacheManager, cs *watch.Controlle
 		return nil, fmt.Errorf("cacheManager must be non-nil")
 	}
 
+	r, err := newStatsReporter()
```

Comment:
What are the observability goals for this metric? As in: what kinds of conclusions should consumers be able to draw by observing this metric? Is the data we are expressing sufficient to support those goals?

What are the gaps in reporting present in this current code?

Reply:
If GK admin and consumer of metrics are two different users, then I think having number of kinds getting synced in config resource might be helpful when access to config resource is resticted?